src/cryptonote_core/service_node_quorum_cop.cpp

```diff
@@ -44,6 +44,34 @@
 
 namespace service_nodes
 {
+  char const *service_node_test_results::why(bool testing_myself) const
+  {
+    static char buf[2048];
+    buf[0]              = 0;
+    char *buf_ptr       = buf;
+    char const *buf_end = buf + sizeof(buf);
+
+    if (passed())
+    {
+      buf_ptr += snprintf(buf_ptr, buf_end - buf_ptr, "Service Node is passing all tests, if decommissioned, you will be recommissioned if status is maintained by the next quorum");
```

Comment:
I think we should shorten this to just "Service node is passing all tests".  Otherwise we're going to invite some edge case where someone sees the message but a dereg has already been sent to the mempool, or some test that can't be easily detected is failing, or ...

Reply:
Yes, good idea